packages/devextreme/js/__internal/ui/m_file_uploader.ts

```diff
@@ -834,15 +836,16 @@ class FileUploader extends Editor<FileUploaderProperties> {
       disabled: this.option('readOnly'),
       hoverStateEnabled: this.option('hoverStateEnabled'),
     });
-    this._selectFileDialogHandler = this._selectButtonClickHandler.bind(this);
+
+    this._selectFileDialogClickHandler = this._selectButtonClickHandler.bind(this);
```

Comment:
I believe we can get rid of this caches if we use namespaces

Reply:
https://github.com/DevExpress/DevExtreme/pull/28947#discussion_r1954548220